operations/products.md

```diff
@@ -29,351 +32,361 @@ Returns all products offered together with the specified services. Note this ope
+    "Cursor": "3fa85f64-5717-4562-b3fc-2c963f66afa6"
+  },
+  "EnterpriseIds": [
+    "3fa85f64-5717-4562-b3fc-2c963f66afa6"
+  ],
+  "ProductId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+  "FirstTimeUnitStartUtc": "2024-04-11T08:54:47.193Z",
+  "LastTimeUnitStartUtc": "2024-04-11T08:54:47.193Z"
 }
 ```
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
-| `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the [Enterprises](enterprises.md#enterprise). If not specified, the operation returns data for all enterprises within scope of the Access Token. |
 | `Client` | string | required | Name and version of the client application. |
-| `ProductId` | string | required | Unique identifier of the [Product](#product) whose prices should be returned. |
-| `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first time unit, in UTC timezone ISO 8601 format. See [Time units](../concepts/time-units.md). |
-| `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last time unit, in UTC timezone ISO 8601 format. See [Time units](../concepts/time-units.md). The maximum size of time interval depends on the service's time unit: 100 hours if hours, 100 days if days, or 24 months if months. |
+| `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the Enterprises. If not specified, the operation returns data for all enterprises within scope of the Access Token. |
+| `ProductId` | string | required | Unique identifier of the product. |
+| `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first [time unit](../concepts/time-units.md), in UTC timezone ISO 8601 format. |
+| `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last [time unit](../concepts/time-units.md), in UTC timezone ISO 8601 format. The maximum size of time interval depends on the service's [time unit](../concepts/time-units.md): 100 hours if hours, 100 days if days, or 24 months if months. |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of data returned. |
 
 ### Response
 
 ```javascript
 {
-    "ProductId": "6b97a38b-0043-41e0-afbd-3f083bdbc0d2",
-    "TimeUnitStartsUtc": [
-        "2024-03-01T23:00:00Z",
-        "2024-03-02T23:00:00Z",
-        "2024-03-03T23:00:00Z"
-    ],
-    "BaseAmountPrices": [
+  "ProductId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+  "TimeUnitStartsUtc": [
+    "string"
+  ],
+  "BaseAmountPrices": [
+    {
+      "Currency": "string",
+      "NetValue": 0,
+      "GrossValue": 0,
```

Comment:
Is it ok that it is "string" and zeros? I may have missed what we agreed on this.

Reply:
I assume It's because the endpoint wasn't present when the data was put into enrich.
@jnv I guess I will have to manually change and then it should be fine for next generations as well?

Reply:
Solved